.circleci/config.yml

```diff
@@ -33,12 +33,16 @@ jobs:
               docker login -u "$DOCKER_REGISTRY_USER" -p "$DOCKER_REGISTRY_PASSWORD" quay.io
               docker push "quay.io/weaveworks/flux:$(docker/image-tag)"
             fi
+            if echo "${CIRCLE_TAG}" | grep -Eq "helm-^[0-9]+(\.[0-9]+)*(-[a-z]+)?$"; then
```

Comment:
Shouldn't this be:

"^helm-[0-9]+(\.[0-9]+)*(-[a-z]+)?$"

?

Reply:
oops, thanks

Reply:
Still needs changing